canvas_manage_course/views.py

```diff
@@ -80,17 +78,15 @@ def lti_launch(request):
 @login_required
 @lti_permission_required('canvas_manage_course')
 def dashboard_course(request):
-    course_instance_id = request.LTI.get('lis_course_offering_sourcedid')
```

Comment:
Not sure this should be deleted

Reply:
The variable is unused in this method. The subsequent method need it, the  dashboard doesnt use it.